Module/STIG/Convert/Functions.XccdfXml.ps1

```diff
@@ -421,16 +443,17 @@ function Get-RuleChangeLog
     {
         $id = $change.Groups.Item('id').value
         $oldText = $change.Groups.Item('oldText').value
-        # The trim removes any potential CRLF enties that will show up in a regex escape sequence.
-        $newText = $change.Groups.Item('newText').value.Trim()
+        # The trim removes any potential CRLF enties that will show up in a regex escape sequence. 
+        #The replace replaces `r`n with an actual new line. This is useful if you need to add data on a separate line.
```

Comment:
[Style guidelines ](https://github.com/PowerShell/DscResources/blob/master/StyleGuidelines.md#correct-format-for-comments)say there should be a space after the # for comments